api/capi/src/nnstreamer-capi-single.c

```diff
@@ -748,32 +758,49 @@ ml_single_invoke (ml_single_h single,
   in_data = (ml_tensors_data_s *) input;
   *output = NULL;
 
-  if (!single_h->filter || single_h->state >= JOIN_REQUESTED) {
+  if (!single_h->filter) {
```

Comment:
You need to understand what's going on with this code and rewrite the error message. The current error message doesn't look accurate.